docs/source/Compiler.md

```diff
@@ -132,6 +132,11 @@ Additional options:
     std::string from Flatbuffers, but (char* + length). This allows efficient
 	construction of custom string types, including zero-copy construction.
 
+-   `--cpp-std CPP_STD` : Generate a C++ code using features of selected C++ standard.
+     Supported `CPP_STD` values:
+    * `c++` - use a standard code generation rules;
```

Comment:
Why the name `c++` for this? Seems ambiguous, especially if this changes with future features/standards.

The old standard we had was effectively C++0x (some, but not all C++11 features), and the new features you're using are C++17. Why not have the names reflect that? That way, if we want to introduce C++14 or C++20 specific features, we have an easy way to do that. They could even be numbers for easy comparison: 0 (legacy), 11, 14, 17 and 20.

Reply:
Yes, `c++` is ambiguous, C++11 will be better.
Replace `legacy` by `C++0x`?

Reply:
Yes, sounds good.